packages/angular_devkit/build_angular/src/dev-server/schema.json

```diff
@@ -49,12 +49,12 @@
     },
     "liveReload": {
       "type": "boolean",
-      "description": "Whether to reload the page on change, using live-reload.",
+      "description": "Whether to reload the page on change, using live-reload with the public host.",
       "default": true
     },
     "publicHost": {
       "type": "string",
-      "description": "Specify the URL that the browser client will use."
```

Comment:
It seems like this should be the same as https://github.com/jbogarthyde/angular-cli/blob/master/packages/angular/cli/lib/config/schema.json#L1098-L1101